templates/definition/meter/sungrow-inverter.yaml

```diff
@@ -48,4 +48,27 @@ render: |
       address: 5003 # Total power yields, 1 kWh
       type: input
       decode: uint32s
+  currents:
```

Comment:
Should only do this for grid, not for pv?

Reply:
It's a possible for PV as well and the data is there. That's the last missing checkmark. It's more of a completionist thing 

The grid one will come after #5314 is merged.

Reply:
We‘re not reading it for pv. no point adding.

Reply:
<img width="489" alt="image" src="https://user-images.githubusercontent.com/5090631/204732672-d4ba3e89-0cdc-4714-8c2d-1dfed610482f.png">

I've added it because it is possible. Yes we don't use it but it also doesn't hurt. I'll close the PR for now.